src/oc/hoc.c

```diff
@@ -721,10 +730,10 @@ void hoc_execerror_mes(const char* s, const char* t, int prnt){	/* recover from
 		hoc_win_normal_cursor();
 #endif
 	if (hoc_oc_jmpbuf) {
-		hoc_newobj1_err();
+		hoc_newobj1_err((void*)2);
```

Comment:
I am not enough familiar with the code but wondering about wha it means to typecast int as void*

Reply:
I'm using void* as a kind of handle identifier that is only used for equality comparison with another handle identifier. 0x1 and 0x2 distinguish the two jmp_buf usages in hoc.c and are assumed never to be equal to a OcJumpImpl pointer from ivoc/ocjump.cpp. there can be many instances of the latter and (void*) has a domain sufficient to cover pointers. What you see here is just to be signature consistent with the earlier ```extern void hoc_newobj1_err(void* jmp);``.  But at a higher level you are right and this is legacy from which the implementation evolved so that no arg is needed decide what function to call to determine which jmp_buf handle to compare with the handles on the stack. I'm making that change.